package.json

```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.15.16",
+    "@react-native-community/netinfo": "^5.9.9",
```

Comment:
why are we using old version

Reply:
The latest version of the package is not supported with the current RN version of our project.